docs/source/auth/index.rst

```diff
@@ -0,0 +1,613 @@
+.. _auth:
```

Comment:
This looks fantastic!

Reply:
I'm going to file a PR to allow setting up credentials in `~/.mlflow/credentials` :) We might need a small adjustment once that's merged.

Reply:
Opened #8688

Reply:
thanks! :) I'll add the missing part